src/index.js

```diff
@@ -16,7 +16,7 @@
  * @typedef Options
  * @property {string} [url] the URL to request
  * @property {'get'|'post'|'put'|'patch'|'delete'|'options'|'head'|'GET'|'POST'|'PUT'|'PATCH'|'DELETE'|'OPTIONS'|'HEAD'} [method="get"] HTTP method, case-insensitive
- * @property {Headers} [headers] Request headers
+ * @property {RequestHeaders} [headers] Request headers
```

Comment:
I believe this is actually called `HeadersInit` in dom.d.ts?

```suggestion
 * @property {HeadersInit} [headers] Request headers
```